EXTENSIONS.md

```diff
@@ -5,7 +5,7 @@ To leverage these extensions, see [How to use](README.md#how-to-use) in `README.
 
 | Extension name  |  Description  |
 | --------------- | ------------- |
-| [azure-chaos-fn-webapp-startstop](https://github.com/trstringer/azure-chaos-fn-webapp-startstop)  | Starts and stops Azure web apps.  |
+| [platform-chaos-fn-webapp-startstop](https://github.com/trstringer/platform-chaos-fn-webapp-startstop)  | Starts and stops Azure web apps.  |
```

Comment:
This url has been changed to https://github.com/trstringer/platform-chaos-fn-startstop